Cybersecurity_Tools/Behavior-Based Script Detector/examples/test_malicious.py

```
@@ -0,0 +1,171 @@
+
+import os
+import subprocess
+import base64
+import pickle
+import marshal
+import socket
+import urllib.request
+import requests
+import threading
+import multiprocessing
+import ctypes
```

Ruff (0.12.2)

`ctypes` imported but unused

Remove unused import: `ctypes`

(F401)